### DISCUSSION TITLE
Login with access token

### DISCUSSION THREAD

**Discussion by sommye-ctr** [supabase/supabase#14649] [Questions]
I want to create a custom auth solution and for it I would be required to retrieve access token of a user and use it for logging in to a new device.

For example, a user already has an account and wants to add a device to the account. Instead of logging in again to the new device, he should be able to do it by any other handy mechanism.

One way I thought was to use the user's access token  so Is this possible? If not, is there any other way I can go about this (ofc cannot use user's password without encrypting it)

**Comment by GaryAustin1** [comment]
I'm moving this to discussion as this is not a bug/issue with the product, but looking for input on a solution to a specific problem.

**Comment by jijkbird** [comment]
@sommye-ctr Have you found a solution?
